gdk/common/GithubUtils.py

```diff
@@ -0,0 +1,10 @@
+import requests
+
+
+class GithubUtils:
+    def get_latest_release_name(self, owner, repository):
```

Comment:
Nit: You can add a comment here that we typically name the testing framework release by its version.